pkg/httputil/mock/response_writer.go

```diff
@@ -0,0 +1,47 @@
+package mock
```

Comment:
WDYT about putting this file under `pkg/testutils/mock_response_writer.go` instead of introducing new test subpackage?

Reply:
With new package we can call it like `mock.ResposneWriter` so it's obvious it's mock with testutils package it need to be in name